flake.nix

```
@@ -33,7 +33,7 @@
       # TODO: Unlocking the input and pointing to official repository requires
       # updating the patch:
       # https://github.com/danth/stylix/pull/224#discussion_r1460339607.
-      url = "github:GNOME/gnome-shell/46.1";
+      url = "github:GNOME/gnome-shell/47.2";
```

Since we are pointing to the latest tag ([`47.2`](https://gitlab.gnome.org/GNOME/gnome-shell/-/tags/47.2)), we might as well track its default (`main`) branch:

```suggestion
      url = "gitlab:GNOME/gnome-shell";
```

---

Our patches are quite likely to fail, or create a broken theme, when a new version is released. Pinning the version means it won't be updated without someone manually verifying that the patches are okay.